### PR TITLE
[HOTFIX] Optimize presto-guide

### PR DESCRIPTION
**Pull request by chenliang613**
Some users made mistake: copy the assemble jar. Add more description to clarify, need to copy many jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2257/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2476/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10515/

File: docs/presto-guide.md [hunk 1]
```diff
@@ -220,7 +220,8 @@ Now you can use the Presto CLI on the coordinator to query data sources in the c
   Secondly: Create a folder named 'carbondata' under $PRESTO_HOME$/plugin and
   copy all jars from carbondata/integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
         to $PRESTO_HOME$/plugin/carbondata
-
+  **NOTE:**  Not copy one assemble jar, need to copy many jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
```
Comment (qiuchenjian) [inline docs/presto-guide.md]:
```suggestion
  **NOTE:**  Don't just copy assemble jar, make sure to copy all jars from integration/presto/target/carbondata-presto-x.x.x-SNAPSHOT
```

**Comment by zzcclp** [comment]
@chenliang613 I think it's better give a 'NOTE' that it's also support presto 0.210+, for example 0.214 version.

File: docs/presto-guide.md (hunk 1 above)
Comment (chenliang613) [reply docs/presto-guide.md]: How about : Not copy the assemble jar, make sure to copy all jars ...

Comment (qiuchenjian) [reply docs/presto-guide.md]: @chenliang613  "Not copy" is oral， suggest  “No need to copy assemble jar” or “”assemble jar is not required“”

**Comment by zzcclp** [comment]
@xubo245 our team had tested with presto 0.214 and @jackylk tested too.

**Comment by xubo245** [comment]
I mean we need test 0.210,0.211,0.212,0.213,.....  when it change to 0.210+

**Comment by zzcclp** [comment]
@xubo245 yup, you are right, but we can't test them one by one, we can note that it also supports presto 0.214, is it ok?

**Comment by ravipesala** [comment]
@zzcclp Can you verify the presto with current master. There are changes related to Hive metastore is done now. So now carbon behaves as a one of the hive supported format in presto. Please check and let us know your feedback.

**Comment by zzcclp** [comment]
@ravipesala NP, another team of our company will test this and the feature of supporting reading stream segment. Will let you know the feedback.

File: docs/presto-guide.md (hunk 1 above)
Comment (xuchuanyin) [reply docs/presto-guide.md]: Why is this NOTE needed? I think line#221 has already made it clear.

Comment (kevinjmh) [reply docs/presto-guide.md]: Is it the folder name can not draw users' attention? I found that a ZIP file is created also. What about asking user to extract the jars in the zip file to a folder under presto

**Comment by xubo245** [comment]
@zzcclp ok, it's better to test more version if having time.

File: docs/presto-guide.md [hunk 2]
```diff
@@ -144,9 +146,7 @@ To run it as a background process.
 To run it in foreground.
 
 ### Start Presto CLI
-```
-./presto
-```
+
```
[the remark this answers is not in the document]
Comment (chenliang613) [reply docs/presto-guide.md]: yes, removed "./presto".
The next script is for start presto server:
./presto --server <coordinator_ip>:8086 --catalog carbondata --schema <schema_name>

File: docs/presto-guide.md (hunk 1 above)
Comment (chenliang613) [reply docs/presto-guide.md]: > Why is this NOTE needed? I think line#221 has already made it clear.

Many community guys made mistakes, this is why add one more note.

**Comment by sraghunandan** [comment]
LGTM

**Pull request closed without merging**